src/pages/billing/List.js

```diff
@@ -479,6 +471,19 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
               p(['It may not exist, or you may not have access to it.'])
             ])
           ])],
+        [!selectedName, () => h(CreateNewBillingProjectWizard, {
```

Comment:
We will be changing this to show `CreateNewBillingProjectWizard` if `_.isEmpty(billingProjects)` (and deleting line 504), correct?

Reply:
Yes, that's correct.